packages/web-sockets/src/drivers/wb/server.js

```diff
@@ -0,0 +1,21 @@
+// External dependencies
+const { Server } = require('socket.io');
+
+// Internal dependencies
+const { Logger } = require('@booking-services/shared');
+
+const io = new Server();
```

Comment:
If you are initializing the socket-io server here, how are we going to be able to bind it with fastify?